src/lib/detectInstall.js

```diff
@@ -33,10 +33,10 @@ function detectInstall (context) {
 
   // the plugin we're trying to install
   let plugin = parameters.second
-  
+
   // extract the package name and (optionally) version
   let packageName, packageVersion
-  const versionSepRe = /(?<!^|\/)@/
```

Comment:
lookbehinds need a harmony flag on node versions <= 8.9.0

Reply:
@udaypydi Thanks for the PR! Do the tests pass with `yarn ci:test` ?

Reply:
@jamonholmgren all the tests pass with ``` yarn ci:test ```